src/board/board.ts

```diff
@@ -1,26 +1,31 @@
 import chalk from "chalk";
 import { renderTile, Tile } from "../tile/tile";
+import { checkIfMoveIsAllowed } from "../move/move";
```

Comment:
unused import

Reply:
I removed it and will see later to make eslint warn me of any unused import.